src/layman/layer/qgis/output_srs_test.py

```diff
@@ -0,0 +1,31 @@
+from . import output_srs, util
+from layman import settings, app
+from test import process, process_client
+
+OUTPUT_SRS_LIST = [4326, 3857, 32633, 32634, 3059, 5514]
+
+
+def assert_qgis_output_srs_list(workspace, layer, expected_srs_list):
+    with app.app_context():
+        assert util.get_layer_wms_crs_list_values(workspace, layer) == set(expected_srs_list)
+
+
+def test_ensure_output_srs_for_all():
```

Comment:
Consider merging with output_srs_list_test

Reply:
Test that Layman start with and without SLD style does not create QGIS project file